gcsweb/cmd/gcsweb/templates.go

```diff
@@ -105,12 +105,25 @@ func htmlContentHeader(out io.Writer, dirname, path string) error {
 	return tmplContentHeader.Execute(out, args)
 }
 
-const tmplContentFooterText = `</ul>`
+const tmplContentFooterText = `</ul>
+<details>
+	<summary style="display: list-item; padding-left: 1em">Download</summary>
```

Comment:
The minified yahoo/pure CSS used by this page for some reason sets summary display to "block" and that makes the triangle disappear. Setting the display to list-item fixes that